worker/orca_grader/__main__.py

```diff
@@ -170,7 +170,6 @@ def handle_grading_job(grading_job: GradingJobJSON, image_name: str | None = Non
         # Will need to see source code for json.dump.
         temp_job_file.write(json.dumps(grading_job, default=str))
         temp_job_file.flush()
-        temp_job_file.close()
```

Comment:
Same as change above; I think we need this line for the successful docker cp command.